app/jobs/request_verification_code_for_login_job.rb

```diff
@@ -34,17 +34,7 @@ def perform(email_address: nil, phone_number: nil, locale:, visitor_id:, service
           service_type: multi_tenant_service.service_type
         )
       else
-        url = if service_type == :statefile && state_code.present?
-                StateFile::StateInformationService.navigation_class(state_code)::FLOW.first.to_path_helper(
-                  us_state: state_code,
-                  full_url: true,
-                  host: URI(Rails.configuration.statefile_url).hostname,
-                  locale: locale,
-                  protocol: "https"
-                )
-              else
-                multi_tenant_service.url(locale: locale)
-              end
+        url = multi_tenant_service.url(locale: locale)
```

Comment:
there does seem to be a slight difference in behavior here? `multi_tenant_service.url` returns the root path whereas this used to return the first page in the flow? which admittedly is a weird difference to have between the products but just wanted to make sure we talk about it